### PR TITLE
Feature/explorer buttons

### PR DESCRIPTION
**Pull request by VladMartyshevskyi**
Integrated editor button to ECMS Explorer action bar.

### CODE REVIEW

File: services/src/main/java/org/exoplatform/officeonline/AbstractOfficeOnlineService.java [hunk 1]
```diff
@@ -275,7 +275,7 @@ public String getVersion() throws RepositoryException {
    * @param port the port
    * @return the string builder
    */
-  protected StringBuilder platformUrl(String schema, String host, int port) {
```
Comment (pnedonosko) [inline services/src/main/java/org/exoplatform/officeonline/AbstractOfficeOnlineService.java]: It should stay protected as it's only a helper, not something service level.

File: services/src/main/java/org/exoplatform/officeonline/documents/OfficeOnlineDocumentEditorPlugin.java [hunk 2]
```diff
@@ -204,16 +308,33 @@ protected String getEditorLink(Node node, URI requestURI) {
...
+                             .toString();
+      } else {
+        throw new OfficeOnlineException("Cannot get editor link - request URI and PortalRequestContext are null");
       }
-      return null;
-    });
+    }
+    String link = null;
+    if (wopiService.canEdit(node)) {
+      link = wopiService.getEditorLink(node, baseUrl, WOPIService.EDIT_ACTION);
```
Comment (pnedonosko) [inline services/src/main/java/org/exoplatform/officeonline/documents/OfficeOnlineDocumentEditorPlugin.java]: Let's pass all parts if the URL to the service without using platformUrl() here which should not be a public method.

**Review by pnedonosko — CHANGES_REQUESTED** [review]

File: services/src/main/java/org/exoplatform/officeonline/documents/OfficeOnlineDocumentEditorPlugin.java (hunk 2 above)
Comment (VladMartyshevskyi) [reply services/src/main/java/org/exoplatform/officeonline/documents/OfficeOnlineDocumentEditorPlugin.java]: Fixed: https://github.com/exo-addons/office-online/pull/31/commits/ca043163690bc4c751e447a58539aceb6e4ce310

File: services/src/main/java/org/exoplatform/officeonline/AbstractOfficeOnlineService.java (hunk 1 above)
Comment (VladMartyshevskyi) [reply services/src/main/java/org/exoplatform/officeonline/AbstractOfficeOnlineService.java]: Fixed: https://github.com/exo-addons/office-online/pull/31/commits/ca043163690bc4c751e447a58539aceb6e4ce310

**Review by pnedonosko — APPROVED** [review]

**Pull request merged**